config/sqlite3.sql

```diff
@@ -55,6 +55,11 @@ CREATE TABLE "mt_metadata" (
   "property_value" text NOT NULL,
   CONSTRAINT "mt_metadata_idfk1" FOREIGN KEY ("item_id") REFERENCES "mt_cds_object" ("id") ON DELETE CASCADE ON UPDATE CASCADE
 );
+CREATE TABLE "mt_config_value" (
```

Comment:
Would be `grb_` nowadays surely ;)

Reply:
I thought about it already, wanted to keep the naming scheme. But, it's not a bad idea to flag our additions correctly. I'll push an update.

Reply:
Just experienced an issue that the server hang when a database command caused an exception. Reason behind: The sqliteThread was still running and the exception could not break out.